src/metrics.rs

```diff
@@ -0,0 +1,338 @@
+use serde::Deserialize;
+
+use {
+  log::*,
+  std::collections::HashMap,
+  std::fmt,
+  std::sync::{atomic, Arc, Mutex, RwLock},
+  tokio::time,
+  warp::{Filter, Rejection, Reply},
+};
+
+#[derive(Debug)]
+enum Value {
+  U64 {
+      value: Arc<atomic::AtomicU64>,
+      metric_type: MetricType,
+  },
+  I64 {
+      value: Arc<atomic::AtomicI64>,
+      metric_type: MetricType,
+  },
+  Bool {
```

Comment:
I think we can avoid this code. We do not use prometheus currently, and if we use it later then we can do how it is done in lite-rpc.